stl/inc/format

```diff
@@ -586,7 +586,8 @@ constexpr void _Parse_format_string(basic_string_view<_CharT> _Format_str, _Hand
                 // In this case we didn't find either a closing curl or opening curl.

```

Comment:
Pre-existing: `_STD find` on 582 could also be `_Find_unchecked`. We know that `[_Begin, _End)` is valid because we got `_Begin` and `_End` from a range (`basic_string_view`), and we know that `[_Begin + 1, _End)` is valid because we checked that `_Begin != _End` on 578. (We only need the extra validation/checking in `_STD find` when the iterators come from an untrusted source, i.e., a user ;).)